lib/src/mapbox_map.dart

```diff
@@ -222,14 +222,6 @@ class MapboxMap extends StatefulWidget {
   /// * All fade/transition animations have completed
   final OnMapIdleCallback? onMapIdle;
 
-  /// Set `MapboxMap.useHybridComposition` to `false` in order use Virtual-Display
-  /// (better for Android 9 and below but may result in errors on Android 12)
-  /// or leave it `true` (default) to use Hybrid composition (Slower on Android 9 and below).
-  static bool get useHybridComposition =>
```

Comment:
might be better to set this to deprecated and remove all functionality, with a note stating that it is no longer required.